styles.css

```diff
@@ -1,38 +1,637 @@
+}
+
+h2 {
+  font-size: 1.5rem;
+  /* font-weight: 500; */
+}
+
+h3 {
+  font-size: 1.2rem;
+  /* font-weight: 500; */
+}
+
+h4 {
+  font-size: 1rem;
+  /* font-weight: 500; */
+}
+
+h5 {
+  font-size: 0.8rem;
+  /* font-weight: 500; */
+}
+
+h6 {
+  font-size: 0.6rem;
+  /* font-weight: 500; */
+}
+
+div {
+  display: block;
+}
+
+ul {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  padding-inline-start: 40px;
+}
+
+a {
+  margin: 0.25rem;
+  text-decoration: none;
+}
+
+pre {
+  display: inline-block;
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+.m-1rem {
+  margin: 1rem;
+}
+
+.m-2rem {
+  margin: 2rem;
+}
+
+.p-1rem {
+  padding: 1rem;
+}
+
+.p-2rem {
+  padding: 2rem;
+}
+
+.title {
+  margin-right: 1.7rem;
 }
 
 .container {
-  color: #fff;
-  position: absolute;
+  max-width: 80rem;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: 300px 1fr;
+}
+
+.sidebar {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  padding: 2rem;
+  border-right: 1px solid #e1e1e1;
+  border-radius: 0.5rem;
+  border-bottom: 0;
+  height: 100vh;
+  position: sticky;
+  top: 0;
+  /* padding: 0.5rem; */
+  z-index: 1;
+  /* background-color: var(--white); */
+  /* box-shadow: rgb(149 157 165 / 20%) 0px 8px 24px; */
+}
+
+.sidebar-header {
   display: flex;
-  flex-direction: column;
+  justify-content: space-between;
   align-items: center;
-  margin: 10rem auto auto 25rem;
 }
 
-.container h1 {
+.fa-cuttlefish {
+  position: absolute;
+  left: 1rem;
+  top: 0.3rem;
   font-size: 4rem;
+  /* color: #fff; */
 }
 
-.documentation-anchor {
-  color: black;
-  border: 1px solid white;
-  text-decoration: none;
-  padding: 0.5rem 1rem;
+.sidebar-content {
+  max-height: initial;
+  visibility: visible;
+  overflow-y: auto;
+  opacity: 1;
+  width: 16rem;
+}
+
+.intro-section {
+  min-height: 40vh;
+  text-align: center;
+}
+
+.intro-div {
+  display: block;
+  margin: 5rem auto 1rem;
+  align-items: center;
+}
+
+.ui-div {
+  border: solid #dee2e6;
+  width: fit-content;
+  margin-left: 10rem;
+}
+
+.list-item {
+  cursor: pointer;
+}
+
+.component-nav-list {
+  list-style: none;
+  padding-left: 0;
+}
+
+.nav-link {
+  display: block;
+  color: inherit;
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+}
+
+.nav-link:hover {
+  font-weight: bold;
+}
+
+.content {
+  padding: 1.5rem;
+  /* width: 60rem; */
+}
+
+.row {
+  padding: 1rem;
+  border-radius: 0.5rem;
+  background-color: var(--white);
+  margin: auto auto 1.6rem;
+  box-shadow: rgb(0 0 0 / 10%) 0px 4px 12px;
+}
+
+.content-div-flex {
+  display: flex;
+  flex-wrap: wrap;
+  border: solid #dee2e6;
+  padding: 1rem;
+  width: fit-content;
+}
+
+.content-div {
+  border: solid #dee2e6;
+  box-sizing: border-box;
+  padding: 1rem;
+}
+
+.width-fit {
+  width: fit-content;
+}
+
+.color-block {
+  width: 80px;
+  height: 50px;
+  border-radius: 5px;
+  margin: 0.8rem;
+}
+
+.primary-color {
+  color: white;
```

Comment:
#### Here, `color: white` can be added in `: root` variable, so that in the future changing in one place will reflect all over in the code.